tests/pipelines/test_pipelines_common.py

```diff
@@ -182,69 +88,166 @@ def __repr__(self):
         return f"ANY({', '.join(_type.__name__ for _type in self._types)})"
 
 
+def is_test_to_skip(test_casse_name, config_class, model_architecture, tokenizer_name, processor_name):
+    """Some tests are just not working"""
+
+    to_skip = False
+
+    if config_class.__name__ == "RoCBertConfig" and test_casse_name in [
```

Comment:
This and below is making me a bit uncomfortable/hard to maintain. Ok for now, but let's fix those that we can fix in a follow-up PR.